app/src/main/res/layout/activity_set_password.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/window_background"
+    android:gravity="center_horizontal"
+    android:orientation="vertical">
+
+    <LinearLayout
```

Comment:
We should try to avoid using containers, and go over `ConstraintLayout` I assume you are re-using some UI from other screen for velocity sake, but let's keep that in mind.